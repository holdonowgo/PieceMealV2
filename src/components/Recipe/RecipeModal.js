import React, { Component } from 'react';
import { Modal } from 'react-native';
import { Content } from 'native-base';
import RecipeView from './RecipeView';

export default class ModalExample extends Component {

  state = {
    modalVisible: false,
  }

  setModalVisible(visible) {
    this.setState({ modalVisible: visible });
  }

  render() {
    return (
      <Content style={{ marginTop: 22 }}>
        <Modal
          animationType={'slide'}
          transparent={false}
          // visible={this.state.modalVisible}
          visible={this.props.visible}
          onRequestClose={() => { alert('Modal has been closed.'); }}
        >
        <RecipeView {...this.props} />
        </Modal>
      </Content>
    );
  }
}
