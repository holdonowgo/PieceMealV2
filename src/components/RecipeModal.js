import React, { Component } from 'react';
// import { Body, Button, Icon, Image, View, Card, CardItem, Thumbnail, Text, Left } from 'native-base';
import { Image, Modal, TouchableHighlight } from 'react-native';
import { Container, Content, Card, CardItem, Thumbnail, Text, Button, Icon, Left, Body, View } from 'native-base';
import space_rockets from '../assets/space_rockets.jpg';
import sliders from '../assets/food/sliders.jpg';

export default class ModalExample extends Component {

  state = {
    modalVisible: false,
  }

  setModalVisible(visible) {
    this.setState({ modalVisible: visible });
  }

  render() {
    return (
      <View style={{ marginTop: 22 }}>
        <Modal
          animationType={"slide"}
          transparent={false}
          // visible={this.state.modalVisible}
          visible={this.props.visible}
          onRequestClose={() => { alert("Modal has been closed."); }}
          >
         <View style={{ marginTop: 22 }}>
          <View>
            <Card style={{ flex: 0 }}>
              <CardItem>
                  <Left>
                      <Thumbnail source={this.props.image} />
                      <Body>
                          <Text>{this.props.name}</Text>
                          <Text note>April 15, 2016</Text>
                      </Body>
                  </Left>
              </CardItem>
              <CardItem>
                  <Body>
                      <Image style={{ flex: 1, height: 150, width: 150 }}
                        source={space_rockets}
                        resizeMode="contain" />
                      <Text>
                          {this.props.description}
                      </Text>
                      <Button transparent textStyle={{color: '#87838B'}}>
                          <Text>1,926 stars</Text>
                      </Button>
                  </Body>
              </CardItem>

              <TouchableHighlight onPress={() => {
                this.props.setModalVisible(!this.props.visible)
              }}>
              <Button onPress={() => {
                this.props.setModalVisible(!this.props.visible)
              }}><Text>Close</Text></Button>
             </TouchableHighlight>
             </Card>

          </View>
         </View>
        </Modal>
      </View>
    );
  }
}
