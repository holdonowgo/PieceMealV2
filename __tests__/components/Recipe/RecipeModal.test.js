import 'react-native';
import React from 'react';

import RecipeModal from '../../../src/components/Recipe/RecipeModal';
import renderer from 'react-test-renderer';

describe('tests the RecipeHeader component', () => {
  it('renders recipeHeader properly', () => {
    expect(renderer.create(
      <RecipeModal
        name={'name'}
      />
    )).toMatchSnapshot();
  });
});
