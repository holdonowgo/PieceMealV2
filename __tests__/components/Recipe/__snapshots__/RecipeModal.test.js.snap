// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests the RecipeHeader component renders recipeHeader properly 1`] = `
<RCTScrollView
  automaticallyAdjustContentInsets={false}
  contentInset={
    Object {
      "bottom": 0,
    }
  }
  enableAutoAutomaticScroll={true}
  enableResetScrollToCoords={true}
  extraHeight={75}
  extraScrollHeight={0}
  keyboardDismissMode="interactive"
  onScroll={[Function]}
  resetScrollToCoords={
    Object {
      "x": 0,
      "y": 0,
    }
  }
  scrollEventThrottle={0}
  showsVerticalScrollIndicator={true}
  style={
    Object {
      "backgroundColor": "transparent",
      "flex": 1,
      "marginTop": 22,
    }
  }
  virtual={undefined}
>
  <View>
    <Modal
      animationType="slide"
      hardwareAccelerated={false}
      onRequestClose={[Function]}
      transparent={false}
      visible={true}
    >
      <RCTScrollView
        automaticallyAdjustContentInsets={false}
        contentInset={
          Object {
            "bottom": 0,
          }
        }
        enableAutoAutomaticScroll={true}
        enableResetScrollToCoords={true}
        extraHeight={75}
        extraScrollHeight={0}
        keyboardDismissMode="interactive"
        onScroll={[Function]}
        resetScrollToCoords={
          Object {
            "x": 0,
            "y": 0,
          }
        }
        scrollEventThrottle={0}
        showsVerticalScrollIndicator={true}
        style={
          Object {
            "backgroundColor": "transparent",
            "flex": 1,
          }
        }
        virtual={undefined}
      >
        <View>
          <View
            style={
              Object {
                "backgroundColor": "#fff",
                "borderColor": "#ccc",
                "borderRadius": 2,
                "borderWidth": 0.5,
                "elevation": 3,
                "flex": 1,
                "flexWrap": "wrap",
                "marginBottom": 5,
                "marginLeft": 2,
                "marginRight": 2,
                "marginTop": 5,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.1,
                "shadowRadius": 1.5,
              }
            }
            virtual={undefined}
          >
            <View
              header={true}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "backgroundColor": "#fff",
                  "borderBottomWidth": null,
                  "flexDirection": "row",
                  "paddingBottom": 15,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 15,
                }
              }
              virtual={undefined}
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "color": "#000",
                    "fontFamily": "Futura",
                    "fontSize": 24,
                    "fontWeight": "500",
                    "marginTop": 20,
                    "paddingTop": 15,
                  }
                }
                virtual={undefined}
              >
                NAME
              </Text>
            </View>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#fff",
                  "flexDirection": "row",
                  "paddingBottom": 10,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 10,
                }
              }
              virtual={undefined}
            >
              <View
                style={
                  Object {
                    "alignItems": "flex-start",
                    "alignSelf": "stretch",
                    "flex": 1,
                  }
                }
                virtual={undefined}
              >
                <Image
                  source={undefined}
                  style={
                    Object {
                      "height": 200,
                      "resizeMode": "contain",
                      "width": 340,
                    }
                  }
                />
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  note={true}
                  style={
                    Object {
                      "alignSelf": "center",
                      "color": "#808080",
                      "fontFamily": "Futura",
                      "fontSize": 14,
                      "fontWeight": "200",
                      "lineHeight": 20,
                      "marginBottom": 10,
                      "marginLeft": 20,
                      "marginRight": 15,
                      "marginTop": 15,
                    }
                  }
                  virtual={undefined}
                >
                  "
                  "
                </Text>
              </View>
            </View>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "alignSelf": "center",
                  "color": "#000",
                  "fontFamily": "Futura",
                  "fontSize": 21,
                  "lineHeight": 22,
                }
              }
              virtual={undefined}
            >
              Instructions:
            </Text>
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#fff",
                  "flexDirection": "row",
                  "paddingBottom": 10,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 10,
                }
              }
              virtual={undefined}
            >
              <ActivityIndicator
                animating={true}
                color="#45D56E"
                hidesWhenStopped={true}
                size="large"
                style={
                  Object {
                    "height": 80,
                  }
                }
                virtual={undefined}
              />
            </View>
            <View
              footer={true}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#fff",
                  "borderBottomWidth": null,
                  "flexDirection": "row",
                  "paddingBottom": 25,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 10,
                }
              }
              virtual={undefined}
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                note={true}
                style={
                  Object {
                    "color": "#808080",
                    "fontFamily": "Futura",
                    "fontSize": 14,
                    "fontWeight": "200",
                  }
                }
                virtual={undefined}
              />
            </View>
            <View
              footer={true}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#fff",
                  "borderBottomWidth": null,
                  "flexDirection": "row",
                  "paddingBottom": 10,
                  "paddingLeft": 15,
                  "paddingRight": 15,
                  "paddingTop": 10,
                }
              }
              virtual={undefined}
            >
              <View
                style={
                  Object {
                    "alignSelf": "flex-start",
                    "backgroundColor": "black",
                    "borderRadius": 13.5,
                    "height": 27,
                    "paddingBottom": 3,
                    "paddingLeft": 6,
                    "paddingRight": 6,
                    "paddingTop": 3,
                  }
                }
                virtual={undefined}
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "white",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "lineHeight": 19,
                      "paddingLeft": 3,
                      "paddingRight": 3,
                      "textAlign": "center",
                    }
                  }
                  virtual={undefined}
                >
                  vegetarian
                </Text>
              </View>
              <View
                style={
                  Object {
                    "alignSelf": "flex-start",
                    "backgroundColor": "black",
                    "borderRadius": 13.5,
                    "height": 27,
                    "paddingBottom": 3,
                    "paddingLeft": 6,
                    "paddingRight": 6,
                    "paddingTop": 3,
                  }
                }
                virtual={undefined}
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "white",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "lineHeight": 19,
                      "paddingLeft": 3,
                      "paddingRight": 3,
                      "textAlign": "center",
                    }
                  }
                  virtual={undefined}
                >
                  vegan
                </Text>
              </View>
              <View
                style={
                  Object {
                    "alignSelf": "flex-start",
                    "backgroundColor": "black",
                    "borderRadius": 13.5,
                    "height": 27,
                    "paddingBottom": 3,
                    "paddingLeft": 6,
                    "paddingRight": 6,
                    "paddingTop": 3,
                  }
                }
                virtual={undefined}
              >
                <Text
                  accessible={true}
                  allowFontScaling={true}
                  ellipsizeMode="tail"
                  style={
                    Object {
                      "color": "white",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "lineHeight": 19,
                      "paddingLeft": 3,
                      "paddingRight": 3,
                      "textAlign": "center",
                    }
                  }
                  virtual={undefined}
                >
                  dairy
                </Text>
              </View>
            </View>
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "height": 1334,
              }
            }
            virtual={undefined}
          >
            <RCTScrollView
              automaticallyAdjustContentInsets={false}
              contentInset={
                Object {
                  "bottom": 0,
                }
              }
              enableAutoAutomaticScroll={true}
              enableResetScrollToCoords={true}
              extraHeight={75}
              extraScrollHeight={0}
              keyboardDismissMode="interactive"
              onScroll={[Function]}
              resetScrollToCoords={
                Object {
                  "x": 0,
                  "y": 0,
                }
              }
              scrollEventThrottle={0}
              showsVerticalScrollIndicator={true}
              style={
                Object {
                  "backgroundColor": "transparent",
                  "flex": 1,
                }
              }
              virtual={undefined}
            >
              <View>
                <View
                  style={
                    Object {
                      "alignItems": "center",
                      "bottom": 20,
                      "flexDirection": "column",
                      "height": 0,
                      "left": undefined,
                      "marginBottom": 30,
                      "marginLeft": 20,
                      "marginTop": 0,
                      "position": "absolute",
                      "right": 20,
                      "top": undefined,
                      "width": 56,
                    }
                  }
                >
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "flex-start",
                        "backgroundColor": "blue",
                        "borderColor": "#007aff",
                        "borderRadius": 20,
                        "borderWidth": null,
                        "bottom": 0,
                        "elevation": 2,
                        "flexDirection": "row",
                        "height": 40,
                        "justifyContent": "center",
                        "left": -50,
                        "marginBottom": 10,
                        "opacity": 1,
                        "paddingBottom": null,
                        "paddingLeft": null,
                        "paddingRight": null,
                        "paddingTop": null,
                        "position": "absolute",
                        "right": 0,
                        "top": 8,
                        "width": 40,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={false}
                      ellipsizeMode="tail"
                      style={
                        Array [
                          Object {
                            "color": undefined,
                            "fontSize": 12,
                          },
                          Object {
                            "alignSelf": "center",
                            "color": "#fff",
                            "fontSize": 24,
                            "marginLeft": 5,
                            "marginRight": 5,
                            "paddingTop": 2,
                          },
                          Object {
                            "fontFamily": "Ionicons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                        ]
                      }
                      virtual={undefined}
                    >
                      
                    </Text>
                  </View>
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "flex-start",
                        "backgroundColor": "#3B5998",
                        "borderColor": "#007aff",
                        "borderRadius": 20,
                        "borderWidth": null,
                        "bottom": 0,
                        "elevation": 2,
                        "flexDirection": "row",
                        "height": 40,
                        "justifyContent": "center",
                        "left": -100,
                        "marginBottom": 10,
                        "opacity": 1,
                        "paddingBottom": null,
                        "paddingLeft": null,
                        "paddingRight": null,
                        "paddingTop": null,
                        "position": "absolute",
                        "right": 0,
                        "top": 8,
                        "width": 40,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={false}
                      ellipsizeMode="tail"
                      style={
                        Array [
                          Object {
                            "color": undefined,
                            "fontSize": 12,
                          },
                          Object {
                            "alignSelf": "center",
                            "color": "#fff",
                            "fontSize": 24,
                            "marginLeft": 5,
                            "marginRight": 5,
                            "paddingTop": 2,
                          },
                          Object {
                            "fontFamily": "Ionicons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                        ]
                      }
                      virtual={undefined}
                    >
                      
                    </Text>
                  </View>
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "flex-start",
                        "backgroundColor": "#DD5144",
                        "borderColor": "#007aff",
                        "borderRadius": 20,
                        "borderWidth": null,
                        "bottom": 0,
                        "elevation": 2,
                        "flexDirection": "row",
                        "height": 40,
                        "justifyContent": "center",
                        "left": -150,
                        "marginBottom": 10,
                        "opacity": 1,
                        "paddingBottom": null,
                        "paddingLeft": null,
                        "paddingRight": null,
                        "paddingTop": null,
                        "position": "absolute",
                        "right": 0,
                        "top": 8,
                        "width": 40,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={false}
                      ellipsizeMode="tail"
                      style={
                        Array [
                          Object {
                            "color": undefined,
                            "fontSize": 12,
                          },
                          Object {
                            "alignSelf": "center",
                            "color": "#fff",
                            "fontSize": 24,
                            "marginLeft": 5,
                            "marginRight": 5,
                            "paddingTop": 2,
                          },
                          Object {
                            "fontFamily": "Ionicons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                        ]
                      }
                      virtual={undefined}
                    >
                      
                    </Text>
                  </View>
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "flex-start",
                        "backgroundColor": "#DD5144",
                        "borderColor": "#007aff",
                        "borderRadius": 20,
                        "borderWidth": null,
                        "bottom": 0,
                        "elevation": 2,
                        "flexDirection": "row",
                        "height": 40,
                        "justifyContent": "center",
                        "left": -200,
                        "marginBottom": 10,
                        "opacity": 1,
                        "paddingBottom": null,
                        "paddingLeft": null,
                        "paddingRight": null,
                        "paddingTop": null,
                        "position": "absolute",
                        "right": 0,
                        "top": 8,
                        "width": 40,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={false}
                      ellipsizeMode="tail"
                      style={
                        Array [
                          Object {
                            "color": undefined,
                            "fontSize": 12,
                          },
                          Object {
                            "alignSelf": "center",
                            "color": "#fff",
                            "fontSize": 24,
                            "marginLeft": 5,
                            "marginRight": 5,
                            "paddingTop": 2,
                          },
                          Object {
                            "fontFamily": "Ionicons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                        ]
                      }
                      virtual={undefined}
                    >
                      
                    </Text>
                  </View>
                  <View
                    accessibilityComponentType={undefined}
                    accessibilityLabel={undefined}
                    accessibilityTraits={undefined}
                    accessible={true}
                    hitSlop={undefined}
                    isTVSelectable={true}
                    onLayout={undefined}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "alignItems": "center",
                        "backgroundColor": "#5067FF",
                        "borderRadius": 28,
                        "bottom": 0,
                        "elevation": 4,
                        "height": 56,
                        "justifyContent": "center",
                        "opacity": 1,
                        "position": "absolute",
                        "width": 56,
                      }
                    }
                    testID={undefined}
                    tvParallaxProperties={undefined}
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={false}
                      ellipsizeMode="tail"
                      style={
                        Array [
                          Object {
                            "color": undefined,
                            "fontSize": 12,
                          },
                          Object {
                            "color": "#fff",
                            "fontSize": 24,
                            "lineHeight": 27,
                          },
                          Object {
                            "fontFamily": "Ionicons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                        ]
                      }
                      virtual={undefined}
                    >
                      
                    </Text>
                  </View>
                </View>
              </View>
            </RCTScrollView>
            <Modal
              animationType="fade"
              hardwareAccelerated={false}
              onRequestClose={[Function]}
              transparent={true}
              visible={false}
            >
              <View
                style={
                  Object {
                    "flex": 1,
                    "justifyContent": "flex-start",
                    "margin": 20,
                  }
                }
              >
                <View
                  danger={false}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "rgba(0,0,0,0.8)",
                      "borderRadius": 5,
                      "flexDirection": "row",
                      "justifyContent": "space-between",
                      "minHeight": 50,
                      "paddingBottom": 10,
                      "paddingLeft": 10,
                      "paddingRight": 10,
                      "paddingTop": 10,
                    }
                  }
                  success={false}
                  virtual={undefined}
                  warning={false}
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "#fff",
                        "fontFamily": "System",
                        "fontSize": 16,
                      }
                    }
                    virtual={undefined}
                  />
                </View>
              </View>
            </Modal>
            <Modal
              animationType="fade"
              hardwareAccelerated={false}
              onRequestClose={[Function]}
              transparent={true}
              visible={false}
            >
              <View
                accessibilityComponentType={undefined}
                accessibilityLabel={undefined}
                accessibilityTraits={undefined}
                accessible={true}
                hitSlop={undefined}
                isTVSelectable={true}
                onLayout={undefined}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "rgba(0,0,0,0.4)",
                    "flex": 1,
                    "justifyContent": "flex-end",
                    "opacity": 1,
                  }
                }
                testID={undefined}
                tvParallaxProperties={undefined}
              >
                <View
                  accessibilityComponentType={undefined}
                  accessibilityLabel={undefined}
                  accessibilityTraits={undefined}
                  accessible={true}
                  hitSlop={undefined}
                  isTVSelectable={true}
                  onLayout={undefined}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "backgroundColor": "#fff",
                      "elevation": 4,
                      "height": NaN,
                      "opacity": 1,
                      "padding": 15,
                    }
                  }
                  testID={undefined}
                  tvParallaxProperties={undefined}
                >
                  <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                      Object {
                        "color": "#757575",
                        "fontFamily": "System",
                        "fontSize": 16,
                      }
                    }
                    virtual={undefined}
                  />
                  <RCTScrollView
                    dataArray={Array []}
                    dataSource={
                      ListViewDataSource {
                        "items": 0,
                      }
                    }
                    enableEmptySections={true}
                    renderRow={[Function]}
                    renderScrollComponent={[Function]}
                    style={
                      Object {
                        "marginHorizontal": -15,
                        "marginTop": 15,
                      }
                    }
                    virtual={undefined}
                  >
                    <View />
                  </RCTScrollView>
                </View>
              </View>
            </Modal>
          </View>
        </View>
      </RCTScrollView>
    </Modal>
  </View>
</RCTScrollView>
`;
